Model/Security/User.cs

```diff
@@ -13,6 +13,8 @@ public class User
 	[MaxLength(64)] // Same as in AD
 	public string Name { get; set; }
 
+	public string Lastname { get; set; }
```

Comment:
stringové property v modelu musí mít `[MaxLength]` (dle interních konvencích _Havit.Blazor stacku_), i kdyby se tím mělo jen explicitně vyjádřit, že to je `Int32.MaxValue` (`nvarchar(max)`). Zde bych se raději držel nějakého limitu, např. `[MaxLength(64)]`.